jobs/types/job_run_result.py

```diff
@@ -143,27 +152,33 @@ def len_in_bytes(self):
         return sum(len(output) for output in output_bytes)
 
     def __repr__(self):
+        # type: () -> str
         return '%s(stdout=%s, stderr=%s)' % (
             self.__class__.__name__,
             utils.quoted(self.stdout), utils.quoted(self.stderr))
 
     def __hash__(self):
+        # type: () -> int
         return hash((self.stdout, self.stderr))
 
     def __eq__(self, other):
+        # type: (Any) -> Any
```

Comment:
```suggestion
        # type: (Any) -> bool
```

Reply:
Oh, interesting, can we add a comment linking the mypy issue?

Reply:
Done.